definitions/marts/looker_studio_marts/ecf/ls_ecf_provider_activity_schedule_changes.sqlx

```diff
@@ -0,0 +1,117 @@
+config {
+    type: "table",
+    bigquery: {
+        partitionBy: "event_date",
+        clusterBy: ["cpd_lead_provider_name"]
+    },
+    description: "This mart contains successful (200 status code) API requests made by a Lead Provider on the API path ending in 'change-schedule' to change a participants schedule_id on their participant profile. This change can also change a participant's cohort. \n \
+	This mart includes changes where the occurred_at date is in the last 180 days only. \n \
+	The purpose of this mart is to support ECF Contract Management in monitoring Lead Provider made changes, namely, the changing of a participants schedule & possibly cohort. \n \
+	The data within this mart will be used in the ECF Contract Management Dashboard to provide a high level count of changes made by the Lead Provider on an Event Date as well as key information that can assist in their conversations with LPs.",
+    columns: {
+      event_date: "The date the change was made in the service. This is called occurred_at in the source data table and renamed to event_date in the mart.",
+      event_date_delta: "The number of days between the event_date and the current date.",
+      cpd_lead_provider_name: "The name of the lead provider who performed the action. This is determined by the request_user_id which is captured in the service and mapped against the cpd_lead_provider_id in cpd_lead_providers_latest. This connection has been confirmed as correct by LPDOB.",
+	  request_uuid: "The unique identifier for the API request that triggered the change to the participant's schedule_id. This UUID is only usable by LPDOB or developers if the request needs to be reviewed. In this mart this value is unique and a distinct count of this field is used to determine how many actions of this type have been conducted by a lead provider on the event date.",
+	  participant_profile_id: "The participant profile ID of the participant whose schedule has been changed.",

```

Comment:
did the LP provide this?

Reply:
The API path contains the user_id so that the correct profile is amended.

Reply:
Can we flag that here? That we get user_id and match on the user_id in the participant profile id?

Reply:
The service handles the API interpretation I believe. I purely look at the changes made to participant_profiles and still derive the user_id from the teacher_profile_id on the profile. Please let me know what are the next steps with this field.